dp-terraform/helm/rhacs-terraform/terraform_cluster.sh

```diff
@@ -79,8 +87,16 @@ if [[ "${OPERATOR_USE_UPSTREAM}" == "true" ]]; then
     OPERATOR_SOURCE="rhacs-operators"
 fi
 
-# helm template --debug ... to debug changes
-helm upgrade rhacs-terraform "${SCRIPT_DIR}" \
+helm_upgrade() {
+    declare -a HELM_EXTRA_FLAGS
+    if [[ "${DEBUG:-}" == "true" ]]; then
```

Comment:
`$DEBUG` is very generic and I wouldn't expect it to imply dry-run too. How about something like:

```suggestion
    if [[ "${HELM_PRINT_ONLY:-}" == "true" ]]; then
```